examples/bch-example/src/app/bch.virtualAccount.example.ts

```diff
@@ -7,34 +7,55 @@ export async function bchVirtualAccountExample() {
 
   const bchSDK = TatumBchSDK({ apiKey: '75ea3138-d0a1-47df-932e-acb3ee807dab' })
 
-  // Check whether a blockchain address is assigned to a virtual account
-  // You can find more details in https://apidoc.tatum.io/tag/Blockchain-addresses#operation/addressExists
-  const account = await bchSDK.virtualAccount.depositAddress.checkExists(
-    'qzla5ev0pap3yrvr0t77k4sjerhr8l3a6sh98gqqya',
-  )
-  console.log(account)
+  // if you don't already have a wallet, address and private key - generate them
+  // You can find more details in https://apidoc.tatum.io/tag/Bitcoin-Cash#operation/BchGenerateWallet
+  const { mnemonic, xpub } = await bchSDK.wallet.generateWallet()
+
+  // Generate PrivateKey from Mnemonic with a given index
+  // You can find more details in https://apidoc.tatum.io/tag/Bitcoin-Cash#operation/BchGenerateAddressPrivateKey
+  const fromPrivateKey = await bchSDK.wallet.generatePrivateKeyFromMnemonic(mnemonic, 0)
+
+  // Generate Address from xpub with a given index
+  // You can find more details in https://apidoc.tatum.io/tag/Bitcoin-Cash#operation/BchGenerateAddress
+  const to = bchSDK.wallet.generateAddressFromXPub(xpub, 1)
+
+  // Generate new virtual account for BCH with specific blockchain address
+  // You can find more details in https://apidoc.tatum.io/tag/Account#operation/createAccount
+  const virtualAccount = await bchSDK.ledger.account.create({
+    currency: 'BCH',
+    xpub: xpub,
+  })
+  console.log(JSON.stringify(virtualAccount))
 
   // Create a deposit address for a virtual account
   // You can find more details in https://apidoc.tatum.io/tag/Blockchain-addresses#operation/generateDepositAddress
-  const address = await bchSDK.virtualAccount.depositAddress.create('5e68c66581f2ee32bc354087', 1)
+  const address = await bchSDK.virtualAccount.depositAddress.create(virtualAccount.id)
   console.log(address)
 
-  //Create multiple deposit addresses for a virtual account
+  // Fund your address here: https://faucet.fullstack.cash/
+  console.log(`Fund me ${address.address} to send virtual account transaction!`)
+
+  // Create multiple deposit addresses for a virtual account
   // You can find more details in https://apidoc.tatum.io/tag/Blockchain-addresses#operation/generateDepositAddressesBatch
   const addresses = await bchSDK.virtualAccount.depositAddress.createMultiple({
     addresses: [
       {
-        accountId: '5e6be8e9e6aa436299950c41',
+        accountId: virtualAccount.id,
         derivationKey: 0,
       },
       {
-        accountId: '5e6be8e9e6aa436299951n35',
+        accountId: virtualAccount.id,
         derivationKey: 1,
       },
     ],
   })
   console.log(addresses)
 
+  // Check whether a blockchain address is assigned to a virtual account
```

Comment:
there is missing send tx from virtual account here

Reply:
added.